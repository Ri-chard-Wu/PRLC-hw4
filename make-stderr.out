hw4.cu(95): warning: missing return statement at end of non-void function "decode"

hw4.cu(112): warning: expression has no effect

hw4.cu(112): warning: expression has no effect

hw4.cu(362): warning: integer conversion resulted in truncation

hw4.cu(396): warning: variable "j" was declared but never referenced

hw4.cu(520): warning: integer conversion resulted in truncation

hw4.cu(648): warning: variable "nlz_target" was declared but never referenced

hw4.cu(95): warning: missing return statement at end of non-void function "decode"

hw4.cu(112): warning: expression has no effect

hw4.cu(112): warning: expression has no effect

hw4.cu(362): warning: integer conversion resulted in truncation

hw4.cu(520): warning: integer conversion resulted in truncation

