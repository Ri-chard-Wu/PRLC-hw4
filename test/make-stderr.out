main.cu(175): warning: variable "gtid" was declared but never referenced

